src/test/java/org/fundacionjala/coding/jorge/SortTwisted37Test.java

```diff
@@ -0,0 +1,50 @@
+package org.fundacionjala.coding.jorge;
+
+import org.junit.Test;
+import org.junit.Assert;
+
+/**
+ *
+ */
+public class SortTwisted37Test {
+    private static final int NUM_1 = 1;
+    private static final int NUM_2 = 2;
+    private static final int NUM_3 = 3;
+    private static final int NUM_4 = 4;
+    private static final int NUM_5 = 5;
+    private static final int NUM_6 = 6;
+    private static final int NUM_7 = 7;
+    private static final int NUM_8 = 8;
+    private static final int NUM_9 = 9;
+    private static final int NUM_12 = 12;
+    private static final int NUM_13 = 13;
+    private static final int NUM_14 = 14;
```

Comment:
@AT-08/at-08 
Don't create constants for each number.
Please declare your arrays as final inside test method.